src/backend/data/schema.js

```diff
@@ -66,23 +67,25 @@ const authRequired = (session) => {
       message: 'You must login to access that resource.'
     });
   }
-}
+};
 
 const adminRequired = (session) => {
   if (!session.user || !session.user.isAdmin) {
     throw new GraphQLError({
-      status: 404,
-      message: 'Nothing here.'
+      status: 401,
```

Comment:
If we go this route, probably a 403 makes more sense (though I realize this is all a bit arbitrary anyway and I'm co-opting HTTP status codes in a world where they aren't actually being used in the same way).